runtimes/parachain/src/lib.rs

```diff
@@ -278,6 +278,7 @@ impl did::Config for Runtime {
 	type WeightInfo = ();
 }
 
+#[allow(clippy::from_over_into)]
```

Comment:
is this doing something? Otherwise i would remove it?

Reply:
That is right, I hoped it would solve the clippy issue, but I will remove it.

Reply:
Fixed in 51333a7